src/it/java/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/NoWhitespaceBeforeTest.java

```diff
@@ -63,5 +63,21 @@ public void testColonOfLabel() throws Exception {
         verify(checkConfig, filePath, expected, warnList);
     }
 
+    @Test
+    public void testAnnotationsAndEllipsis() throws Exception {
+        final Class<NoWhitespaceBeforeCheck> clazz = NoWhitespaceBeforeCheck.class;
+        final String messageKeyPreceded = "ws.preceded";
+
+        final String[] expected = {
+            "12:32: " + getCheckMessage(clazz, messageKeyPreceded, "..."),
+        };
+
+        final Configuration checkConfig = getModuleConfig("NoWhitespaceBefore");
```

Comment:
I think this checkConfig is not proper for google style, but I do not find a way to add attribute to config like https://github.com/checkstyle/checkstyle/blob/11bb48abe8bff4c8c7597de5c667ea2ca935475e/src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceBeforeCheckTest.java#L76

Reply:
you do not need to update Test to add attribute, if you need such token - add it to google_checks.xml file.
Such tests do validation of google config as is, without any modification.
Such tests are not tests for Check, they are tests for exact config.

Reply:
done, I get it